docs/guide/deployment/resource-directory.md

```diff
@@ -234,6 +234,7 @@ This will be served by HTTP Gateway API as defined [here](https://github.com/plg
 | `publicConfiguration.cloudID` | string | `Cloud ID which is stored in coap-gw certificate.` | `""` |
 | `publicConfiguration.cloudURL` | string | `Cloud URL for onboard device.` | `""` |
 | `publicConfiguration.cloudAuthorizationProvider` | string | `OAuth authorization provider for onboard device.` | `""` |
+| `publicConfiguration.defaultCommandTTL` | string | `Exposes default command time to live for CreateResource, RetrieveResource, UpdateResource, DeleteResource, and UpdateDeviceMetadata commands when it is zero value in the request. 0s - means forever.` | `"0s"` |
```

Comment:
I was a bit confused by the "when it is zero value in the request." and the subsequent "0s - means forever." Since both have zero in it so I read it together, but it didn't make much sense. :D

Maybe reformulating it to "when it is not set in the request" would be better?

Reply:
@rafajpet  ping